src/blocks/stars.css

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  border-radius: 0.5px;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: 50%;
+}
+
+.stars__star:last-child {

```

Comment:
quantity of stars can be changeable, copy logic from stars task

Reply:
Did you mean to put .stars__star:nth-child(-n+4) instead :last-child ?

Reply:
I meant to copy logic from [this](https://github.com/mate-academy/layout_stars) task

Reply:
copied logic from star task